src/components/sideNav.js

```diff
@@ -41,40 +45,72 @@ export default function sideNav(props) {
                   Title
                   TotalContributions
                 }
+                Referendums {
+                  Title
+                  Description
+                  Total_Contributions
+                }
               }
             }
           }
         }
       `}
       render={data => {
-        const { Date, OfficeElections } = data.allElection.edges[0].node
-        const menu = formatMenu(OfficeElections)
-        const sections = Object.keys(menu)
+        let navLinks = null;
+        if (window.location.href.includes('referendum')) {
```

Comment:
Do we not have any information about the currently selected item here that we can use instead? This seems a little bit fragile